### DISCUSSION TITLE
Paperless NGX Script Problem with tmp folder in Debian lxc, when using consume directory

### DISCUSSION THREAD

**Discussion by knorst1** [tteck/Proxmox#2374] [Help Needed]
Hi, 
first of all, thank you for the good work!
I noticed, that something is faulty, when I use the Paperless NGX -helper script and try to copy files to the consume directory of Paperless NGX inside the running LXC-container.
it somehow cannot create the directory /tmp/paperless/.. for the OCR-process (no such file or directory!) . So it's not possible to add files via the consume directory. 
if i create the the directory "/tmp/paperless" manually with mkdir paperless in /tmp, the files are processing normally. the problem is: with every reboot of the lcx, the /tmp directory in debian will be erased/cleaned...so with every new reboot of the LXC-container, this problem occurs.
Any ideas? Problem of the Helper-script or is it a Paperless NGX-Problem?
greetings
knorst1

**Comment by AlexGeek77** [comment]
What if you don't use /tmp ? That folder is not meant to hold permanent files or folders.

**Reply by knorst1** [reply]
i tried it, but didn't succeed. i could not find any option to change the settings. also the files are stored in the temp folder only while processing, then moved to the archive folder from paperless...
